Update DM TOC and remove redundant files (#7493)
pingcap/docs: dm/feature-expression-filter.md

---
title: Filter DMLs Using SQL Expressions
aliases: ['tidb/dev/feature-expression-filter/']
---

# Filter DMLs Using SQL Expressions

## Overview

> In the process of incremental data migration, you can filter certain types of binlog events using the [Filter Binlog Events](/filter-binlog-event.md) feature. For example, for archiving or auditing purposes, `DELETE` event might be filtered when data is migrated to the downstream. However, Binlog Event Filter cannot judge with a greater granularity whether the `DELETE` event of a certain row should be filtered.

Reviewer: ```suggestion
In the process of incremental data migration, you can filter certain types of binlog events using the [Filter Binlog Events](/filter-binlog-event.md) feature. For example, for archiving or auditing purposes, you can filter out `DELETE` events when migrating data to the downstream. However, Binlog Event Filter cannot judge with a greater granularity on whether to filter out a specific row of `DELETE` events.
```
